x/asset/keeper/pairs_vault.go

```diff
@@ -302,15 +302,16 @@ func (k Keeper) VaultIterateRewards(ctx sdk.Context, collectorLsr sdk.Dec, colle
 			if !found {
 				continue
 			}
+
 			var interest sdk.Dec
 			var err error
 			if vaultData.BlockHeight == 0 {
-				interest, err = k.rewards.CalculationOfRewards(ctx, vaultData.AmountOut, collectorLsr, collectorBt)
+				_, _ = k.rewards.CalculationOfRewards(ctx, vaultData.AmountOut, collectorLsr, collectorBt)
```

Comment:
this change can't be accepted. interest, err has to be there else it breaks the below code and line 324 interest will always be the default value.